### PR TITLE
Object storage final polish

### PR DESCRIPTION
**Pull request by eddie-knight**
Hopefully this wraps up Object Storage to make it ready for a release candidate.

Follows #418 to reduce the diff

### CODE REVIEW

File: services/common-controls.yaml [hunk 1]
```diff
@@ -117,14 +165,34 @@ controls:
...
+        text: |
+          The service logs all access attempts from untrusted entities,
+          including failed connection attempts.
+        tlp_levels:
+          - tlp_amber
+          - tlp_red
+      - id: CCC.C05.TR04
+        text: |
+          The service prevents unauthorized cross-tenant access, ensuring that
+          only allowlisted services from other tenants can access resources.
```
Comment (mlysaght2017) [inline services/common-controls.yaml:188]: what do we mean by "tenant" here? I think that's worth expanding on.

**Review by mlysaght2017 — APPROVED** [review]
Great polish @eddie-knight - LGTM! (one comment on the meaning of "tenant")

**Review by damienjburks — APPROVED** [review]

File: services/common-controls.yaml (hunk 1 above)
Comment (eddie-knight) [reply services/common-controls.yaml]: This change isn't in the current PR after merging 418, but I think it's worth spinning back around to if you'd like to propose a change to common controls

File: services/storage/object/controls.yaml [hunk 2]
```diff
@@ -10,14 +10,13 @@ controls:
...
-      Prevent any requests to object storage buckets or objects using untrusted
-      KMS keys to protect against unauthorized data encryption that can impact
-      data availability and integrity.
+      Prevent any requests to object storage buckets or objects using 
+      untrusted KMS keys to protect against unauthorized data encryption 
+      that can impact data availability and integrity.
     control_family: Data
     threats:
       - CCC.TH01
       - CCC.TH02
```
Comment (sshiells-scottlogic) [inline services/storage/object/controls.yaml:19]: Should we add comments here like we do with common features etc?

**Review by sshiells-scottlogic — APPROVED** [review]

File: services/storage/object/controls.yaml (hunk 2 above)
Comment (eddie-knight) [reply services/storage/object/controls.yaml:19]: Yeah, that's a good idea. I'll aim for another PR with that, but it shouldn't be a blocker for cutting the release candidate.

**Pull request merged**